File: kubernetes/minikube/plugins/js/traceheader-js/schema.js
module.exports = {
  name: "trace-header-js",
  fields: [
    {
      config: {
        type: "record",
        fields: [
          {
            trace_header: {
              type: "string",
              default: "X-Trace-ID",
              required: true,
            },
          },
        ],
      },
    },
  ],
};

File: kubernetes/minikube/plugins/lua/traceheader-lua/schema.lua
return {
    name = "trace-header-lua",
    fields = {
        {
            config = {
                type = "record",
                fields = {
                    { trace_header = { type = "string", default = "X-Trace-ID", required = true }, },
                },
            },
        },
    }
}
